openedx/core/djangoapps/programs/utils.py

```diff
@@ -285,7 +285,7 @@ def completed_programs_with_available_dates(self):
         Returns a dict of {uuid_string: available_datetime}
         """
         # Query for all user certs up front, for performance reasons (rather than querying per course run).
-        user_certificates = GeneratedCertificate.eligible_certificates.filter(user=self.user)
+        user_certificates = GeneratedCertificate.eligible_available_certificates.filter(user=self.user)
```

Comment:
Does a test cover this? Probably not necessary.

Reply:
yea I think the model tests effectively cover this